Jewish Calendar: Document Services (#31770)
home-assistant/home-assistant.github.io: source/_integrations/jewish_calendar.markdown

  diaspora: true
  havdalah_minutes_after_sunset: 50
```

## Services

The Jewish Calendar integration also includes four services that return Hebrew calendar and Zmanim information about specific day(s).

### Common Attributes

> All of these services return the same fields, based on which of the following `include_*` attributes you pass to the service. If you set multiple attributes to true, it will return all of thise fields.

Reviewer: ```suggestion
These services return the same fields, based on the following `include_*` attributes. If you set multiple attributes to true, they will return all these fields.
```

Author: Done
